src/parameters/parameters_flow_solver.cpp

```diff
@@ -75,10 +83,30 @@ void FlowSolverParam::declare_parameters(dealii::ParameterHandler &prm)
         prm.declare_entry("output_restart_files_every_dt_time_intervals", "0.0",
                           dealii::Patterns::Double(0,dealii::Patterns::Double::max_double_value),
                           "Outputs the restart files at time intervals of dt.");
-      
-        prm.declare_entry("input_mesh_filename", "naca0012",
-                          dealii::Patterns::FileName(dealii::Patterns::FileName::FileType::input),
-                          "Filename of the input mesh: input_mesh_filename.msh");
+
+        prm.enter_subsection("grid");
```

Comment:
I think its more natural for the flow solver to use these parameters from its parameters section rather than `grid_refinement_study` since that section is dedicated to a grid refinement study and we're not doing anything related to that

Reply:
Okay, that is fair. It leads to a longer list of parameters, but I don't see any downsides otherwise. In the future we could potentially change to have a unified `parameters_grid_generation` or similar where all grid generation is stored (but not refinement or flow-solver-specific), but the repetition is not an issue right now.

Reply:
Yeah that's a good idea, I think if we eventually introduced a mesh factory as part of a class called `GridGeneration` then that parameter section would be great